packages/firebase_messaging/firebase_messaging/example/pubspec.yaml

```diff
@@ -6,7 +6,7 @@ environment:
   flutter: '>=1.12.13+hotfix.5'
 
 dependencies:
-  firebase_core: ^2.5.0
+  firebase_core: ^2.6.0
   firebase_messaging: ^14.2.2
```

Comment:
https://github.com/firebase/flutterfire/pull/10472/files#diff-4d9a8918a1b3e8d9549bae0e9973c779ca6bd2b771b1ebb7f9791c2d761d4b70R6 should be updated.